README.md

```diff
@@ -36,6 +35,18 @@ wget https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/c
 kubectl apply -f ./kubemacpool.yaml
 ``` 
 
+### Opting-in to kubemacpool service
```

Comment:
Please keep this readable even in plain text. Keep one empty line below and above each heading.